codice/ddf
# DDF-3456 Updates Content Directory Monitor documentation

#### What does this PR do?
Updates CDM documentation to clarify a few confusing points. 
Looking for feedback on how I can begin to clarify this even more. 

Do I get some kind of prize for this ticket #? 

#### Who is reviewing it? 
@emmberk @tbatie @ahoffer @garrettfreibott 
@ricklarsen **(Documentation)**

#### Choose 2 committers to review/merge the PR. 
@brendan-hofmann
@clockard

#### How should this be tested? 
Verify the documentation work links when built.
Verify that this documentation is accurate with the current functionality.

#### What are the relevant tickets?
[DDF-3456](https://codice.atlassian.net/browse/DDF-3456)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
